sorted_words.rb

```diff
@@ -0,0 +1,17 @@
+def ask()
+    words = []
+    print "Please enter a word. "
+    answer = gets.chomp
+    words.push(answer)
+    while words.length > 0
```

Comment:
This check is redundant because the condition always holds; an item is inserted in the line above, and at no point do items get removed from the array.

```suggestion
    loop do
```